js/src/degausser.js

```diff
@@ -1,22 +1,40 @@
 import { StringCollector } from './stringCollector'
 import { MapCollector } from './mapCollector'
 import { walkDOM } from './domWalker'
+import { DOMParser } from '@xmldom/xmldom'
```

Comment:
we should use this xmldom library in order to make Degausser work within the Go tool environment?

Reply:
It is a breaking change

Reply:
If this makes things so much harder, maybe we should consider running a Headless Chrome instead https://itnext.io/scrape-the-web-faster-in-go-with-chromedp-c94e43f116ce. 
This should get us having the same results on the server and browser environments.
later we will be able to apply the same approach in NodeJS version.

Reply:
I'm not sure if chromedp is exactly what we need. I didn't find any references to Dom or DomParser within their [API references](https://pkg.go.dev/github.com/chromedp/chromedp#section-readme)

Reply:
> we should use this xmldom library in order to make Degausser work within the Go tool environment?
Yes, we need DomParser. Which we cannot access without xmldom.

Reply:
I'm not really sure which interactions this chromedp allows to make with the Chrome browser. I'm proposing to launch Chrome and interact with it to run the degausser script there, read its output and put it into the Blob